docs/toc.md

```diff
@@ -32,84 +32,84 @@
     - [Build your workers]({{urlRoot}}/content/build)
     - [Deploy your game]({{urlRoot}}/content/deploy)
 - <h3> + GDK concepts</h3>
-    - <h4>Workers: </h4>
-        - [ * Workers in the GDK]({{urlRoot}}/content/workers/workers-in-the-gdk)
-        - [ * Connecting to SpatialOS Runtime]({{urlRoot}}/content/connecting-to-spatialos)
-        - [ * API - worker]({{urlRoot}}/content/workers/api-worker)
-        - [ * API - worker system]({{urlRoot}}/content/workers/api-worker-system)
-    - <h4>SpatialOS entities:</h4>
-        - [ * Update entity lifecycle]({{urlRoot}}/content/entity-lifecycle)
-        - [* Creating SpatialOS entity templates]({{urlRoot}}/content/entity-templates)
-        - [* API - EntityBuilder]({{urlRoot}}/content/api-entity-builder)
-    - <h4>Commands:</h4>
-        - [ * World and component command requests and responses]({{urlRoot}}/content/world-component-commands-requests-responses)
+    - Workers
+        - [Workers in the GDK]({{urlRoot}}/content/workers/workers-in-the-gdk)
+        - [Connecting to SpatialOS Runtime]({{urlRoot}}/content/connecting-to-spatialos)
```

Comment:
Your summary says you've removed gerunds, but there are still several around - is that intentional?

Reply:
👍 Pesky blighters. They will be gone.

Reply:
Have removed most of the gerunds - left a few in where they made sense to be. (Troubleshooting, mailing list and a couple of others.)

Reply:
Just thought of another thing: we usually say "the SpatialOS Runtime" - do we want to add "the" in here?

Reply:
Amended.